backends/ebpf/targets/target.py

```diff
@@ -26,8 +26,8 @@
 import os
 import sys
 from glob import glob
-from scapy.utils import rdpcap
-from scapy.layers.all import RawPcapWriter
+from scapy.utils import rdpcap, RawPcapWriter
+from scapy.layers.all import *
```

Comment:
I am not sure this import is actually needed for anything. Seems that `RawPcapWriter` has been in utils.py for a long time.

Reply:
If I comment out either of those lines, the ebpf p4c tests fail.